src/components/control-bar/SeekBar.js

```diff
@@ -108,14 +133,21 @@ export default class SeekBar extends Component {
         getPercent={this.getPercent}
         stepForward={this.stepForward}
         stepBack={this.stepBack}
+        childrenToMerge={this.getDefaultChildren()}
       >
-        <LoadProgressBar
-          buffered={buffered}
-          currentTime={time}
-          duration={duration}
-        />
-        <MouseTimeDisplay duration={duration} mouseTime={mouseTime} />
-        <PlayProgressBar currentTime={time} duration={duration} />
+        {renderGrandchildren ? (
+          children.props.children
+        ) : (
+          <>
+            <LoadProgressBar
+              buffered={buffered}
+              currentTime={time}
+              duration={duration}
+            />
+            <MouseTimeDisplay duration={duration} mouseTime={mouseTime} />
+            <PlayProgressBar currentTime={time} duration={duration} />
+          </>
+        )}
```

Comment:
Non-blocking feedback. Could this be inverted to simplify? That is, `children.props.children` gets passed in a `childrenToMerge` (when `renderGrandchildren` is truthy) and `this.getDefaultChildren()` gets passed as "children"? Or I suppose at that point perhaps `this.getDefaultChildren` is not needed? I imagine this change would also mean you'd want to change the order of arguments when you call `mergeAndSortChildren` in `Slider`, so that the `childrenToMerge` (if they exist) are given priority.